xml/System.Buffers.Binary/BinaryPrimitives.xml

```diff
@@ -452,7 +452,7 @@
       </Parameters>
       <Docs>
         <param name="value">The value to reverse.</param>
-        <summary>Allows reading a struct of numeric primitives and reverse each field rather than having to skip byte fields.</summary>
```

Comment:
I had suggested reverting reverses -> reverse in this case, since this method itself isn't reversing anything. The text is suggesting it **can be** used to reverse the field.

Maybe we can change `... and reverse each field...` to be `... to reverse each field...`
Will defer to you on this as well, @mairaw